### DISCUSSION TITLE
wordmove should ignore or warn about trailing slashes in 'vhost'

### DISCUSSION THREAD

**Discussion by castilma** [welaika/wordmove#644] [Ideas]
$Title, to prevent others from the error in #641.

**Comment by alessandro-fazzi** [comment]
I've updated the doc as a quick way to warn users: https://github.com/welaika/wordmove/wiki/movefile.yml-configurations-explained#vhost

I'll also promote this to a feature request following this [simple idea](https://github.com/welaika/wordmove/issues/641#issuecomment-1038317537) as mandate, thus enabling `wordmove doctor` to warn about the -almost sure- misconfiguration.

Thanks for prompting.

I'll proceed as soon as time will be mercyful ;)

**Comment by alessandro-fazzi** [comment]
Promoted to feature in #650 .
